src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,8 @@
+package duke;
+
+public class DukeException extends IllegalArgumentException {
```

Comment:
Considering inheriting from higher level exception class, as dukeexceptions can consist of other types of exceptions. Alternatively can create more types of dukeExceptions